src/components/NewAccountPage/NewAccountPage.jsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+// import { Button } from 'react-bootstrap';
+// import styled from 'styled-components';
+
+/* const SubmitButton = styled(Button)`
+  background-color: #6593D6;
+  color: white;
+  padding: 10px;
+  border-radius: 10px;
+  border: none;
+`; */
+class NewAccountPage extends React.Component {
```

Comment:
Can you convert this to a functional component?

Reply:
@esauerbo let me know if you need any help with this